### DISCUSSION TITLE
Nginx is not install and remaning command are also not working

### DISCUSSION THREAD

**Discussion by NARENDRA935** [learndevopsonline/learndevopsonline#1684] [Q&A]
Hii @r-devops,
ngnix is not installing and also reamining commands are not working in putty please sove this problem raghu
<img width="930" alt="Screenshot 2023-12-20 190105" src="https://github.com/learndevopsonline/learndevopsonline/assets/153573686/669c35ef-4bc8-478b-b69d-0b36cadc2e49">

**Comment by sreedharm07** [comment]
ping your ip 
you have cropped your ip 
and please dont create multiple discussions 
you are creating multiple
